src/main/java/org/terasology/kcomputers/components/KallistiComputerComponent.java

```diff
@@ -57,4 +44,10 @@ public void onMachineChanged(EntityRef ref) {
             ref.saveComponent(this);
         }
     }
+
+    @Override
+    public void copy(KallistiComputerComponent other) {
+        this.on = other.on;
+        this.machine = other.machine;
```

Comment:
I have no clue whether this is okay for `Machine` :shrug:

Reply:
I've added a `FIXME` here as I don't want to look into creating a deep copy of `Machine` and all it's members right now ... :see_no_evil: